articles/concepts/Glossary.md

```diff
@@ -12,31 +12,33 @@ uid: microsoft.quantum.glossary
-|Pauli Operators|The `X`, `Y` and `Z` quantum gates.|
-|Prelude|The set of primitive and classical operations and functions defined by each individual target machine, rather than at the Q# level.|
-|Quantum Circuit|The representation of a program for a quantum computer. See the <xref:microsoft.quantum.concepts.circuits> section for more details.|
-|Quantum State|A representation of the qubits in the system. This is usually denoted as a complex column vector. For more information, see <xref:microsoft.quantum.concepts.vectors>. |
-|Qubit|Unit of quantum storage. See the <xref:microsoft.quantum.concepts.qubit> section for more details.|
-|Repeat-until-success|A quantum algorithm that probabilistically succeeds. Upon failure, the routine will re-try until successful (or a limit has been reached). |
-|Target machine|A compilation target that lowers an abstract quantum program towards hardware or simulation. This typically include re-writes for many purposes including gate replacement, encoding for error correction, geometric layout and others.|
-|Tuple|Comma separated types grouped together via parenthesis. |
-|User-defined type|Collection of built-in or previously defined types that may be referred to as a single unit.|
-
+|<a id="adjoint"></a>Adjoint|The complex conjugate transpose of an operation. For operations that implement a unitary operator, the adjoint is the inverse of the operation. Adjoints are indicated by dagger symbol, for example the adjoint of vector $v$ is $v^\dagger$. For more information, see [Adjoint](xref:microsoft.quantum.language.file-structure#adjoint).|
+|<a id="bell-state"></a>Bell state|A maximally entangled quantum state of two qubits. Also known as an [EPR pair](xref:microsoft.quantum.glossary#epr)|
+|<a id="callable"></a>Callable|In the Q# language, operations and functions are collectively known as *callables*. For more information, see [Operation and function types](xref:microsoft.quantum.language.type-model#operation-and-function-types).|
+|<a id="clifford"></a>Clifford group|The set of operations that occupy the octants of the Bloch sphere and effect permutations of the Pauli operators. These include: `X`, `Y`, `Z`, `H` and `S`|
+|<a id="controlled"></a>Controlled|A quantum operation that takes one or more qubits as enablers for the target operation. For more information, see [Controlled](xref:microsoft.quantum.language.type-model#controlled).|
+|<a id="dirac"></a>Dirac Notation|A shorthand representation of quantum states, for example $$\frac{1}{\sqrt{2}}\begin{bmatrix} 1 \\\\  1 \end{bmatrix}=H\ket{0} = \ket{+}$$. For more information, see [Dirac Notation](xref:microsoft.quantum.concepts.dirac).|
+|<a id="eigenvalue"></a>Eigenvalue| The factor by which the magnitude of an [eigenvector](xref:microsoft.quantum.glossary#eigenvector) is changed by a given transformation. For more information, see [Advanced matrix concepts](xref:microsoft.quantum.concepts.matrix-advanced).|
```

Comment:
The term "magnitude" often refers to a nonnegative real number, but an eigenvalue can in general be a complex number of any argument (phase).